### PR TITLE
Test bulk creator

### PR DESCRIPTION
**Pull request by goose-life**

### CODE REVIEW

File: indigo/tests/bulk_creator/test_bulk_creator.py [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+from indigo_api.models import Country, Work
+from indigo_app.models import User
+
+
+class BulkCreateWorksTest(testcases.TestCase):
+    fixtures = ['languages_data', 'countries', 'user', 'taxonomies', 'work']
+
+    def setUp(self):
+        self.maxDiff = None
+        creator = plugins.for_locale('bulk-creator', 'za', None, None)
```
Comment (longhotsummer) [inline indigo/tests/bulk_creator/test_bulk_creator.py:20]: Rather create the instance explicitly from the class, so it's clear what's being tested.

Comment (longhotsummer) [reply indigo/tests/bulk_creator/test_bulk_creator.py:20]: If the ZA version is being tested, these tests should be in the tests module for indigo_za (ie. in the same project).

**Review by longhotsummer — CHANGES_REQUESTED** [review]
Nice tests. Please change the test to be explicit about the class that is being tested, and if that must be the ZA importer, then move the tests to be in the same project.

**Comment by goose-life** [comment]
Basic:
image
image

**Comment by goose-life** [comment]
Errors:
image
image

**Comment by goose-life** [comment]
Commencements:
image
image

**Comment by goose-life** [comment]
Amendments:
image
image

image
image

**Comment by goose-life** [comment]
Repeals:
image
image

**Comment by goose-life** [comment]
Parents:
image
image

**Comment by goose-life** [comment]
Duplicates:
image
image

File: indigo_app/templates/indigo_api/work_new_batch.html [hunk 2]
```diff
@@ -108,12 +105,12 @@ <h4 class="card-header">
               <th>Number</th>
               <th>Year</th>
               <th>FRBR URI</th>
+              <th>Taxonomy</th>
               <th>Tasks</th>
               <th>Relationships</th>
-              <th>Actions</th>
+              <th>Notes (shown in Preview only)</th>
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/work_new_batch.html:111]: Minor: I don't think anyone's going to be confused if this is just shown in preview, but not after import. So perhaps the heading can just be "Notes"/"Comments"/"Import notes".

**Review by longhotsummer — APPROVED** [review]

File: indigo_app/templates/indigo_api/work_new_batch.html (hunk 2 above)
Comment (goose-life) [inline indigo_app/templates/indigo_api/work_new_batch.html:111]:
```suggestion
              <th>Notes</th>
```

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**